### PR TITLE
Re-adds parasting, since people seem to want things back the way they were

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: code/game/gamemodes/changeling/powers/tiny_prick.dm [hunk 1]
```diff
@@ -238,4 +238,17 @@
...
+	name = "Paralysis Sting"
+	desc = "We sting a human, causing them to be stunned for a long duration."
+	helptext = "This will not mute them, and they will be able to speak and cry for help."
+	sting_icon = "sting_para"
+	chemical_cost = 1
+	dna_cost = 0
+
+/obj/effect/proc_holder/changeling/sting/cryo/sting_action(mob/user, mob/target)
+	target << "<span class='userdanger'>Your muscles painfully tighten!</span>"
+	target.Weaken(1000000000000000000000)
```
Comment (RemieRichards) [inline code/game/gamemodes/changeling/powers/tiny_prick.dm]: rofl

**Comment by Razharas** [comment]
MEMES
<h1>MEEMEES

**Comment by WJohn** [comment]
Hooo boy. This can't end well.

**Comment by Zilenan91** [comment]
Is this a meme

**Comment by Xhuis** [comment]
No, this is a serious PR. I wanted changeling to be like how it used to be, since it's better. Stealth was actually required and it caused tension.

File: code/game/gamemodes/changeling/powers/tiny_prick.dm (hunk 1 above)
Comment (Jalleo) [inline code/game/gamemodes/changeling/powers/tiny_prick.dm]: That is stupidly long.

**Comment by Jalleo** [comment]
@Xhuis this is a joke pr ya aint reverting it to how it was when it was rem

**Comment by Zilenan91** [comment]
This is a joke. Parasting is shit and always will be shit.

**Comment by Zilenan91** [comment]
And another thing, there is no tension to walking outside your workplace to get some food, feeling a burning sensation, and your round being over.

**Comment by PKPenguin321** [comment]
:laughing:

**Comment by Miauw62** [comment]
Lol

**Pull request closed without merging**